addon/components/base/bs-accordion/item/body.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+import layout from 'ember-bootstrap/templates/components/bs-accordion/body';
+
+/**
+ Component for an accordion item body.
+
+ See [Components.Accordion](Components.Accordion.html) for examples.
+
+ @class AccordionBody
+ @namespace Components
+ @extends Ember.Component
+ @public
+ */
+export default Ember.Component.extend({
+  layout,
+
```

Comment:
for the same reason as the other comment, you can set `tagName: ''` here to make it a tagless component, removing the unnecessary wrapping `<div>`